nixos/modules/services/misc/gitlab.nix

```diff
@@ -673,6 +673,8 @@ in {
         openssh
         nodejs
         gnupg
+        gnutar
```

Comment:
we should add a comment why they're needed (to import gitlab project imports)

Reply:
Does this look okay?

Reply:
Yes, thanks!